LICENSE

```diff
@@ -214,18 +214,22 @@ Apache Software Foundation License 2.0
 3rdparty/dmlc-core
 
 
-BSD License
+BSD 2-clause License
 -----------
 
 3rdparty/picojson
+
+
+BSD 3-clause License
+-----------
+
 3rdparty/dmlc-core/include/dmlc/concurrentqueue.h
```

Comment:
Add 3rdparty/dmlc-core/include/dmlc/blockingconcurrentqueue.h
c.f. https://github.com/apache/incubator-mxnet/blob/master/LICENSE#L337

Reply:
NOTE: we need to add a license file for blockingconcurrentqueue.h, as part of the file is under zlib, and others under BSD-2

Reply:
We should have a section 

BSD 2-clause license + zlib license